dirty.rb

```diff
@@ -0,0 +1,38 @@
+a =     'b'
```

Comment:
__[staging][rubocop]__
* Useless assignment to variable - `a`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)
* Operator `=` should be surrounded by a single space. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Unnecessary spacing detected.